yb-voyager/cmd/importData.go

```diff
@@ -136,6 +136,22 @@ func importDataCommandFn(cmd *cobra.Command, args []string) {
 
 	// Check if target DB has the required permissions
 	if tconf.RunGuardrailsChecks {
+		// If import to source on PG, check if triggers and FKs are disabled
+		fkAndTriggersCheckFailed := false
+		if importerRole == SOURCE_DB_IMPORTER_ROLE {
+			enabledTriggersAndFks, err := tdb.GetEnabledTriggersAndFks()
```

Comment:
Same comment here, Let the function return `(enabledTriggers, enabledFks, err)`. Frame the error message in exportData layer, not targetDB layer.

Reply:
Done